chpl/chpl-service/src/main/java/gov/healthit/chpl/dao/statistics/SummaryStatisticsDAO.java

```diff
@@ -1,14 +1,70 @@
 package gov.healthit.chpl.dao.statistics;
 
+import java.time.LocalDate;
+import java.util.Date;
+import java.util.List;
+
+import javax.persistence.Query;
+
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
+import org.springframework.stereotype.Repository;
+
+import gov.healthit.chpl.auth.user.User;
+import gov.healthit.chpl.dao.impl.BaseDAOImpl;
 import gov.healthit.chpl.entity.SummaryStatisticsEntity;
 import gov.healthit.chpl.exception.EntityCreationException;
 import gov.healthit.chpl.exception.EntityRetrievalException;
 
-public interface SummaryStatisticsDAO {
-    SummaryStatisticsEntity create(SummaryStatisticsEntity summaryStatistics)
-            throws EntityCreationException, EntityRetrievalException;
+@Repository("summaryStatisticsDAO")
+public class SummaryStatisticsDAO extends BaseDAOImpl {
+    private static final Logger LOGGER = LogManager.getLogger(SummaryStatisticsDAO.class);
```

Comment:
Can you switch this to use Log4j2 annotation?

Reply:
Completely removed since it was not being used.